spec/views/ft_forms_mailer/send_confirmation_text_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'ft_forms_mailer/send_confirmation.text.erb' do
+  before :each do
+    @form_data = Hash[question1: 'answer1', email: 'my_email']
```

Comment:
I agree, although I suspect is was the use of the `Hash[]` method which is a bit weird.  That line could also be:

```ruby
@form_data = {question1: 'answer1', email: 'email_address@example.com'}
```